readme.md

```diff
@@ -118,7 +118,8 @@ If your test suite is not exiting it might be because you still have a lingering
 gulp.task('default', () =>
 	gulp.src('test.js')
 		.pipe(mocha())
-		.once('error', () => {
+		.once('error', (err) => {
+			console.error(err.stack);
```

Comment:
In Node 4 (what we're using), `console.error(err)` prints newlines as `'\n'`, instead of advancing the cursor to a new line, which makes it tough to read the stack trace.